css/sponsor-dashboard.css

```diff
@@ -1,4 +1,10 @@
-#fair-heading{
+:root {
```

Comment:
css variables should be in base style file which will be available in all views.